lib/polyfill/storage_estimate.js

```diff
@@ -0,0 +1,54 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.polyfill.StorageEstimate');
+
+goog.require('shaka.polyfill');
+
+
+/**
+ * @summary A polyfill to provide navigator.storage.estimate in old
+ * webkit browsers.
+ */
+shaka.polyfill.StorageEstimate = class {
+  /**
+   * Install the polyfill if needed.
+   */
+  static install() {
+    if ('storage' in navigator && 'estimate' in navigator.storage) {
+      // No need.
+      return;
+    }
+
+    if ('webkitTemporaryStorage' in navigator &&
```

Comment:
Is there any additional fallback that can be done without the prefixed version?  Or do you plan to check existence again before using the API in the library?

Reply:
This is the old API and the final version is navigator.storage.estimate (it is similar to pip_webkit polyfill)